app/services/folio/types.rb

```diff
@@ -66,8 +66,8 @@ def criteria
         'loan-type' => get_type('loan_types').index_by { |p| p['id'] },
         'location-institution' => get_type('institutions').index_by { |p| p['id'] },
         'location-campus' => get_type('campuses').index_by { |p| p['id'] },
-        'location-library' => libraries.all.index_by(&:id),
-        'location-location' => locations.all.index_by(&:id)
+        'location-library' => libraries.all.index_by(&:id).map { |library| { 'code' => library.code } },
+        'location-location' => locations.all.index_by(&:id).map { |location| { 'code' => location.code } }
```

Comment:
I was going to leave https://github.com/sul-dlss/sul-requests/pull/1748 as a code comment, but Folio::Library didn't implement `#to_h`.

I don't know that we want to restrict this to just the code; #1748 keeps in the spirit of the implementation before we extracted classes for that stuff 🤷‍♂️